### DISCUSSION TITLE
Welcome to MQTT-Explorer Discussions!

### DISCUSSION THREAD

**Discussion by thomasnordquist** [thomasnordquist/MQTT-Explorer#787] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Liggisten** [comment]
Love the MQTT explorer, a priceless tool while i try to learn mqtt and python!

I only have one single complaint: i cant seem to get it for my raspberry pi 4, my main development machine.. :(

And when the worst thing you can say about a piece of software is; it works great on most computers, but wont run on my edge-case type of hardware, then you know you got a mighty fine piece of software!

**Reply by bj00rn** [reply]
Under releases you will find versions for your architecture, i assume rpi4 i arm64

https://github.com/thomasnordquist/MQTT-Explorer/releases

**Reply by Liggisten** [reply]
> https://github.com/thomasnordquist/MQTT-Explorer/releases

I completely missed that, so thank you for that link! :)
Its working nearly as flawlessly as on the PC, its a bit of a strain on my Pi4, especially RAM, but thats probably a non-issue on my future Pi5... :)

**Reply by StevoKeano** [reply]
Newbee...  Any ideas?

me:~/Downloads$ uname -m
x86_64
me:~/Downloads$ chmod +x /home/stevo/Downloads/MQTT-Explorer-0.4.0-beta.6-arm64.AppImage
me:~/Downloads$ ./MQTT-Explorer-0.4.0-beta.6-arm64.AppImage
bash: ./MQTT-Explorer-0.4.0-beta.6-arm64.AppImage: cannot execute binary file: Exec format error
